.travis.yml

```diff
@@ -56,14 +56,13 @@ script:
   - export TEST_EXTRA=""
   - if [[ "$COVERAGE" == "true" ]]; then
       export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-    fi
-  - if [[ "$STATIC" == "false" ]]; then
+    elif [[ "$STATIC" == "true" ]]; then
```

Comment:
Is that how you do things?? I usually go by length, putting shorter cases near the top of an if block, and longer cases near the end. That's just a rule of thumb, though, I do make exceptions.